tests/UI/campaigns/functional/BO/02_orders/05_shoppingCarts/01_filterShoppingCarts.js

```diff
@@ -0,0 +1,193 @@
+require('module-alias/register');
+
+// Helpers to open and close browser
+const helper = require('@utils/helpers');
+
+// Common tests login BO
+const loginCommon = require('@commonTests/loginBO');
+
+// Import pages
+const dashboardPage = require('@pages/BO/dashboard');
+const shoppingCartsPage = require('@pages/BO/orders/shoppingCarts');
+
+// Import data
+const {ShoppingCarts} = require('@data/demo/shoppingCarts');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_orders_shoppingCarts_filterShoppingCarts';
+
+// Import expect from chai
+const {expect} = require('chai');
+
+let numberOfShoppingCarts;
+let browserContext;
+let page;
+
+// Today date
+const today = new Date();
+
+// Current day
+const day = (`0${today.getDate()}`).slice(-2);
+
+// Current month
+const month = (`0${today.getMonth() + 1}`).slice(-2);
+
+// Current year
+const year = today.getFullYear();
+
+// Date today format (yyy/mm/dd)
+const todayDate = `${day}/${month}/${year}`;
```

Comment:
This is not coherent. You say in the comment that you want a date with the format `yyyy/mm/dd` but create a date with the format `dd/mm/yyyy` ?

Reply:
Thanks done :+1: